build/build.py

```diff
@@ -444,6 +449,11 @@ def generate_options(self):
         if self.build_for_snap:
             generate_options.extend(["-DDO_BUILD_FOR_SNAP=1"])
 
+        if self.install_prefix:
+            generate_options.extend(["-DCMAKE_PREFIX_PATH={}".format(self.install_prefix)])
+        else:
+            generate_options.extend(["-DCMAKE_PREFIX_PATH=/usr"])
```

Comment:
Let us handle the default in CMakeLists.txt itself. DU agent does not use build.py, so it will help keep the behavior same for them.